src/com/spikes2212/motor/Gearbox.java

```diff
@@ -7,59 +7,60 @@
 
 import java.util.List;
 
-public class Gearbox implements SpeedController {
+public class Gearbox extends WPI_TalonSRX{
 
-    private WPI_TalonSRX master;
     private List<SpeedController> victorSPs;
+    private int port;
 
-    public Gearbox(WPI_TalonSRX master) {
-        this.master = master;
+    public Gearbox(int port) {
```

Comment:
Make the constructor receive a vararg of [`BaseMotorController`](http://www.ctr-electronics.com/downloads/api/java/html/classcom_1_1ctre_1_1phoenix_1_1motorcontrol_1_1can_1_1_base_motor_controller.html#ad4f61f2782b6048c7f905bbf38b066ac) instances, then enslave them using the class method (see also the comment on `enslave`)